tests/integration/test_parsers.py

```diff
@@ -87,6 +87,16 @@
             Path(dir_path, "data", "zayo", "zayo2.html"),
             Path(dir_path, "data", "zayo", "zayo2_result.json"),
         ),
+        (
+            ParserCogent,
+            Path(dir_path, "data", "cogent", "cogent1.html"),
+            Path(dir_path, "data", "cogent", "cogent1_result.json"),
+        ),
+        (
+            ParserCogent,
+            Path(dir_path, "data", "cogent", "cogent2.html"),
```

Comment:
I see a `cogent3.html` file but not used, any reason? if yes, maybe adding note?

Reply:
removed